modules/ROOT/pages/release-notes.adoc

```diff
@@ -54,9 +54,9 @@ Bonita projects layout has been changed to take advantage of Maven multi-module
 
 First project component that will benefit from Maven multi-module project is the Business Data Model. We decided to align the way the Business Data Model is handled in Bonita with the standard dependency handling practices by using Maven submodules. Submodules, or subprojects, are regular Maven projects that inherit from the parent POM.
 
-With the chosen solution, the users will have the possibility to publish and consume the Business Data Model java dependencies independently. The project will pull the required dependencies from a specified repository - Maven Central or Bonita Artifact Repository - to generate the BDM java sources.
+With the chosen solution, the users will have the possibility to publish and consume the xref:data:define-and-deploy-the-bdm.adoc#generated-java-dependencies[Business Data Model java dependencies] independently. The project will pull the required dependencies from a specified repository - Maven Central or Bonita Artifact Repository - to generate the BDM java sources. Here are details on how to configure access to xref:software-extensibility.adoc[Bonita Artifact Repository(Subscription only)] and xref:setup-dev-environment:configure-maven[Maven Central]. 
```

Comment:
What's the link with Bonita Artifact Repository ? It can be confusing as having access to the BAR it not required here.

Reply:
You are right. Went to fast. Thank you for your vigilance!